widget/accordion.go

```diff
@@ -198,6 +200,7 @@ func (r *accordionRenderer) updateObjects() {
 		var h *Button
 		if i < hs {
 			h = r.headers[i]
+			h.Show()
```

Comment:
nit: is this needed?

Reply:
Some times - a header can be hidden, so if it is to be reused after hidden it must be re-shown.